berza/src/main/java/rs/edu/raf/banka/berza/service/impl/OrderService.java

```diff
@@ -2,37 +2,75 @@
 
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
-import rs.edu.raf.banka.berza.dto.OrderDto;
-import rs.edu.raf.banka.berza.enums.HartijaOdVrednostiType;
-import rs.edu.raf.banka.berza.enums.OrderAction;
-import rs.edu.raf.banka.berza.enums.OrderType;
+import rs.edu.raf.banka.berza.enums.*;
 import rs.edu.raf.banka.berza.model.Order;
-import rs.edu.raf.banka.berza.model.UserAccount;
-import rs.edu.raf.banka.berza.repository.AkcijeRepository;
-import rs.edu.raf.banka.berza.repository.ForexRepository;
-import rs.edu.raf.banka.berza.repository.FuturesUgovoriRepository;
 import rs.edu.raf.banka.berza.repository.OrderRepository;
+import rs.edu.raf.banka.berza.response.ApproveRejectOrderResponse;
+import rs.edu.raf.banka.berza.utils.MessageUtils;
 
+import java.util.Date;
 import java.util.List;
 
 @Service
 public class OrderService {
 
     private OrderRepository orderRepository;
+    private FuturesUgovoriPodaciService futuresUgovoriPodaciService;
 
     @Autowired
-    public OrderService(OrderRepository orderRepository, AkcijeRepository akcijeRepository,
-                        ForexRepository forexRepository, FuturesUgovoriRepository futuresUgovoriRepository){
+    public OrderService(OrderRepository orderRepository, FuturesUgovoriPodaciService futuresUgovoriPodaciService){
         this.orderRepository = orderRepository;
+        this.futuresUgovoriPodaciService = futuresUgovoriPodaciService;
     }
 
-    public List<Order> getOrders() {
-        return orderRepository.findAll();
+    public Order getOrder(Long id) {
+        return orderRepository.getById(id);
+    }
+
+    public List<Order> getOrders(String status, Boolean done) {
+        if(status.length() == 0 && done == null)
```

Comment:
Za šta služi ovo `done == null`, šta prezentuje ta vrednost?

Reply:
Mislila sam da se prosledjuje parametar da li zeli filtriranje po statusu i kom, to je opciono. Kao i to da li zeli da izlista samo zavrsene na to se odnosi done. 
Ako imas drugaciju ideju za impl, reci.